eng/Versions.props

```diff
@@ -44,6 +44,7 @@
     <MicrosoftExtensionsDependencyInjectionVersion>10.0.0-preview.3.25166.3</MicrosoftExtensionsDependencyInjectionVersion>
     <MicrosoftExtensionsDependencyInjectionAbstractionsVersion>10.0.0-preview.3.25166.3</MicrosoftExtensionsDependencyInjectionAbstractionsVersion>
     <MicrosoftExtensionsFileProvidersAbstractionsVersion>10.0.0-preview.3.25166.3</MicrosoftExtensionsFileProvidersAbstractionsVersion>
+    <MicrosoftExtensionsHostingAbstractionsVersion>10.0.0-preview.3.25166.3</MicrosoftExtensionsHostingAbstractionsVersion>
```

Comment:
You also need to configure `Version.Details.xml`, adding a new entry for `Microsoft.Extensions.Hosting.Abstractions` like this one:
https://github.com/dotnet/maui/blob/3204eaca3db0470cff4c19813d18cf673f201ddf/eng/Version.Details.xml#L118-L121

This will let the Maestro bot be able to update this version number.